modules/client/examples/05-encoders/06-update-plugin-settings-address-list.ts

```diff
@@ -0,0 +1,32 @@
+/* MARKDOWN
```

Comment:
Wouldn't you make the difference between general DAO actions and plugin-specific actions?

Reply:
LOVE. Will implement!